orca/orca-web/config/orca.yml

```diff
@@ -59,7 +59,7 @@ resilience4j.retry:
     # katoRetrofitServiceWriter is any write operation to Clouddriver using Retrofit.
     katoRetrofitServiceWriter:
       retryExceptions:
-        - retrofit.RetrofitError
+        - com.netflix.spinnaker.kork.retrofit.exceptions.SpinnakerServerException
```

Comment:
I think we'd be better off with this in a separate PR since it's not really related to removing retrofit1.